app/models/sir_trevor_rails/blocks/browse_block.rb

```diff
@@ -34,6 +34,10 @@ def as_json
 
         result[:data][:item] ||= {}
 
+        # TODO: This is a temporary fix that simply removes any item if the search identifier does not exist
+        #       We should have a more permanent solution that will allow browse blocks to be updated without erroring
+        result[:data][:item].select! { |_, v| parent.exhibit.searches.exists?(v['id']) }
```

Comment:
I can think of a few other ways to accomplish this, but this one seemed succinct (particularly since we'll eventually want to remove this line all together).

Happy to discuss alternate approaches.